assets/blocks/quiz/quiz-block/use-update-quiz-has-questions-meta.js

```diff
@@ -32,11 +32,16 @@ export const useUpdateQuizHasQuestionsMeta = ( clientId ) => {
 	const { editPost } = useDispatch( 'core/editor' );
 	const setQuizHasQuestionsMeta = useCallback(
 		( enable ) => {
+			// Check if block is in the DOM (it skips cases in preview iframes).
+			if ( ! document.getElementById( `block-${ clientId }` ) ) {
```

Comment:
I believe this check will never hide again the "View Quiz" button when the quiz block is removed from the page, right?

Testing here, almost every step of the Testing Instructions section pass, except for this one:

> Remove the quiz, and make sure the "View Quiz" is not displayed again.

I believe this is happening because it is looking for the old element of the quiz block represented by `clientId`, and, given that it is not defined, it just exits early here.. :thinking:

Reply:
It worked! Thanks!

Reply:
Thank YOU for the multiple catches! =)